server/middleware/numberValidator.js

```diff
@@ -0,0 +1,33 @@
+import { body } from 'express-validator/check';
+
+/**
+ * Validate meal ${fieldName}
+ *
+ * @param {Object} options - Options object with key optional
+ *
+ * @returns {ValidationChain} - ValidationChain middleware
+ */
```

Comment:
Why is there no description of what this code does? I can barely understand what is going on here. This is not a clean code. You won't last long here.

Reply:
I will add a better description to jsdoc comments to add clarity about what the function does

Reply:
@Nkemjiks LOL